contrib/containers-storage.conf.5.md

```diff
@@ -0,0 +1,61 @@
+% storage.conf(5) Container Storage Configuration File
+% Dan Walsh
+% May 2017
+
+# NAME
+storage.conf - Syntax of Container Storage configuration file
+
+# DESCRIPTION
+The STORAGE configuration file specifies all of the available container storage options
+for tools using shared container storage, but in a TOML format that can be more easily modified
+and versioned.
```

Comment:
The “but…” part seems like something from a PR proposal, not useful for users now that the choice has been made. e.g. “more easily” compared to what?

Reply:
perhaps "... shared container storage in a TOML format that can be modified and versioned."

Reply:
I agree.